profiles/ravithemore.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/87522195/144974395-bad6c53e-1846-48fc-af86-07bf280c9f8f.png",
```

Comment:
I recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/144976103-9e774349-2eee-4cc9-a4eb-6e02d5dabaca.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/144976103-9e774349-2eee-4cc9-a4eb-6e02d5dabaca.png",
```

Reply:
@schmelto, thoughts?